src/meta/src/hummock/manager/tests.rs

```diff
@@ -643,111 +508,19 @@ async fn test_hummock_manager_basic() {
         HummockVersionId::MAX
     );
     for _ in 0..2 {
-        hummock_manager.unpin_version(context_id_1).await.unwrap();
-        assert_eq!(
-            hummock_manager.get_min_pinned_version_id().await,
-            HummockVersionId::MAX
-        );
-
         // should pin latest because u64::MAX
```

Comment:
Please delete this stale comment.

Reply:
just resumed `test_hummock_manager_basic`